package.json

```diff
@@ -4,13 +4,15 @@
   "private": true,
   "main": "public/main.js",
   "dependencies": {
+    "@mdx-js/react": "^2.1.1",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.2.0",
     "@testing-library/user-event": "^13.5.0",
     "electron": "^19.0.1",
     "electron-is-dev": "^2.0.0",
```

Comment:
Move these to `devDependencies`. Likely some of these other packages can be moved there as well.